### ISSUE TITLE
Bluetooth auto trust not working

### ISSUE THREAD

**Issue by Blackbeard25** [nicokaiser/rpi-audio-receiver#23]
bluetooth is discoverable, but when i need to connect a new device i have to open bluetoothctl and trust the new device, otherwise it disconnects after a few seconds. Running Raspbian 9. Tested on a raspberry pi b+, 3b, and 3b+ with the same outcome, any fix?

**Comment by Tim-Dixon** [comment]
Having the same issue. I had to manually turn on bluetooth and make it advertise, then add my device as trusted in order to pair... :/

**Comment by bchavet** [comment]
There are two instances of `/usr/bin/hciconfig` in `/etc/systemd/system/bluetooth-agent.service`. I changed those to `/bin/hciconfig` and that fixed this problem for me

```sh
sed -i s#/usr/bin/hciconfig#/bin/hciconfig# /etc/systemd/system/bluetooth-agent.service
```

**Comment by Blackbeard25** [comment]
that almost fixed it, I ended up just using hciconfig instead of bin/hciconfig to get it working

**Comment by nicokaiser** [comment]
This should be fixed by #24
